src/nupic/regions/tm_region.py

```diff
@@ -453,10 +437,11 @@ def initialize(self):
   #############################################################################
   def compute(self, inputs, outputs):
     """
-    Run one iteration of TMRegion's compute, profiling it if requested.
+    Run one iteration of :class:`~nupic.regions.tm_region.TMRegion` compute, 
+    profiling it if requested.
 
-    The guts of the compute are contained in the _compute() call so that
-    we can profile it if requested.
+    :param inputs: TODO: I need help with this. -- Matt 
+    :param outputs: TODO: I need help with this. -- Matt 
```

Comment:
@scottpurdy Can you or @mrcslws provide descriptions of theses params? The expected data structures need to be defined somewhere.